documentation/src/main/asciidoc/topics/con_performance_data_consistency.adoc

```diff
@@ -0,0 +1,104 @@
+[id='performance-data-consistency_{context}']
+= Data consistency
+
+Data that resides on a distributed system is vulnerable to errors that can arise from temporary network outages, system failures, or just simple human error.
+These external factors are uncontrollable but can have serious consequences for quality of your data.
+The effects of data corruption range from lower customer satisfaction to costly system reconciliation that results in service unavailability.
+
+{brandname} can carry out ACID (atomicity, consistency, isolation, durability) transactions to ensure the cache state is consistent.
+
+Transactions are a sequence of atomic operations that {brandname} caries out as a single operation.
+Either all reads and writes in the transaction complete successfully or they all fail.
```

Comment:
I would remove "reads" from this section. A transaction rollback just discard the writes but nothing happens to the reads (the keys are already read!)